File: backend/ws-service/src/main/java/com/example/wsservice/WsServiceApplication.java
package com.example.wsservice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.cloud.openfeign.EnableFeignClients;

@EnableFeignClients
@SpringBootApplication
@EnableDiscoveryClient
public class WsServiceApplication {

	public static void main(String[] args) {
		SpringApplication.run(WsServiceApplication.class, args);
	}
}

File: backend/ws-service/src/main/kotlin/com/example/wsservice/WsServiceApplication.kt
package com.example.wsservice

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.cloud.openfeign.EnableFeignClients

@EnableFeignClients
@SpringBootApplication
@EnableDiscoveryClient
class WsServiceApplication

fun main(args: Array<String>) {
    runApplication<WsServiceApplication>(*args)
}
